tools-public/toolpad/components/PieChart.tsx

```diff
@@ -16,29 +16,34 @@ const COLORS = [
 ];
 
 export interface PieChartProps {
-  data: object[];
+  data: any[];
+  loading: boolean;
 }
 
-function PieChartExport({ data }: PieChartProps) {
+function PieChartExport({ data, loading }: PieChartProps) {
+  if (loading) {
+    return null;
```

Comment:
Always give feedback. Even if it's just 

```suggestion
    return 'Loading...';
```

Bad looking feedback is better than empty space